tests/integration/test_tzkt_reward_api.py

```diff
@@ -75,13 +76,13 @@ def test_get_rewards_for_cycle_map(self, address, cycle):
                 nw=default_network_config_map['MAINNET'],
                 baking_address=address,
                 node_url='https://rpc.tzkt.io/mainnet')
-            rpc_rewards = rpc_impl.get_rewards_for_cycle_map(cycle)
```

Comment:
@nicolasochem only two reward types are tested here?

Reply:
Yes, I didn't add ideal reward tests to tzkt api.